easybuild/easyconfigs/e/ETSF_IO/ETSF_IO-1.0.4-foss-2019a.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ETSF_IO'
+version = '1.0.4'
+
+homepage = 'http://www.etsf.eu/resources/software/libraries_and_tools'
```

Comment:
@migueldiascosta change to https? also this particular page seems to be down for the moment.